addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PersistentXssPrimeScanRule.java

```diff
@@ -57,12 +57,12 @@ public int getCategory() {
 
     @Override
     public String getSolution() {
-        return Constant.messages.getString(MESSAGE_PREFIX + "misc");
+        return "";
```

Comment:
Does this really need to be done? Can’t the methods just be removed?

Reply:
They need to be implemented.